### PR TITLE
Bakta update to 1.5.0

### PR DESCRIPTION
**Pull request by pimarin**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
You need a rebase here @pimarin

File: tools/bakta/macro.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <macros>
-    <token name="@TOOL_VERSION@">1.4.2</token>
-    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@TOOL_VERSION@">1.5.0</token>
+    <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bgruening) [inline tools/bakta/macro.xml:3]: That needs to be 0 I think

Comment (pimarin) [reply tools/bakta/macro.xml:3]: I don't change the version when I update the tool version also ?

Comment (nsoranzo) [reply tools/bakta/macro.xml:3]: This should be reset to 0 at tool version upgrades, since it's used to track only Galaxy wrapper sub-versions.

**Review by bgruening — APPROVED** [review]

**Pull request merged**